test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+const engine = path.join(__dirname,
+                         `/build/${common.buildType}/testengine.engine`);
+
+const agentKey = fs.readFileSync(path.join(common.fixturesDir,
```

Comment:
Please use the new `require('../common/fixtures')` utility

Reply:
Done